pkg/process/events/listener_linux.go

```diff
@@ -0,0 +1,165 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// Copyright 2016-present Datadog, Inc.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+
+//go:build linux
+// +build linux
+
+package events
+
+import (
+	"context"
+	"errors"
+	"go.uber.org/atomic"
```

Comment:
nit: this should go in the 3rd party import group.

Reply:
Thanks for catching it!
I've fixed it here: https://github.com/DataDog/datadog-agent/pull/12099/commits/3483bcc299ebda145257a11d10995abdb11148af .